bin/reth/src/args/types.rs

```diff
@@ -80,4 +105,19 @@ mod tests {
         let val = "123".parse::<MaxU32>().unwrap();
         assert_eq!(val, MaxU32(123));
     }
+
+    max_values!(TestU32, u32);
```

Comment:
this we don't need, we should only check the generated U32 and U64 structs

Reply:
> this we don't need, we should only check the generated U32 and U64 structs

those are for test block, they aren't generated outside test block, right ?

Reply:
no problem, going to delete them